docs/references/calfits_memo.tex

```diff
@@ -12,177 +13,155 @@
-\item{\textbf{ant\_array}: Array of antenna indices for data arrays, shape
-    (Nants\_data). type = int, 0 indexed}
-\item{\textbf{antenna\_names}: List of antenna names, shape (Nants\_telescope),
-    with numbers given by antenna\_numbers (which can be matched to ant\_array).
-    There must be one entry here for each unique entry in ant\_array, but there may be extras as well.}
-\item{\textbf{antenna\_numbers}: List of integer antenna numbers corresponding
-    to antenna\_names, shape (Nants\_telescope). There must be one entry here for each unique entry 
-    in ant\_array, but there may be extras as well}
-\item{\textbf{channel\_width}: Channel width of of a frequency bin. Units Hz.}
-\item{\textbf{flag\_array}: Array of flags to be applied to calibrated data
-    (logical OR of input and flag generated by calibration). True is flagged.
-    Shape: (Nants\_data, Nspws, Nfreqs, Ntimes, Njones), type = bool.}
-\item{\textbf{freq\_array}: Array of frequencies, shape (Nspws, Nfreqs), units
-    Hz.}
-\item{\textbf{gain\_convention}: The convention for applying the calibration solutions to data.
-    Values are "divide" or "multiply", indicating that to calibrate one should divide or multiply
-    uncalibrated data by gains. Mathematically this indicates the alpha exponent in the equation: 
-    calibrated data = gain$^{\alpha} \times $ uncalibrated data. A value of
-    ``divide'' represents $\alpha=-1$ and ``multiply'' represents $\alpha=1$.}
-\item{\textbf{history}: String of history}
-\item{\textbf{integration\_time}: Integration time of a time bin, units seconds.}
-\item{\textbf{jones\_array}: Array of antenna polarization integers, shape
-    (Njones). linear pols -5:-8 (jxx, jyy, jxy, jyx).circular pols -1:-4 (jrr,
-    jll. jrl, jlr).}
-\item{\textbf{quality\_array}: Array of qualities of calibration solutions. The
-    shape depends on cal\_type, if the cal\_type is "gain" or "unknown" the shape is:
-    (Nants\_data, Nspws, Nfreqs, Ntimes, Njones), if the cal\_type is "delay" the shape is: 
-    (Nants\_data, Nspws, 1, Ntimes, Njones), type = float}
-\item{\textbf{spw\_array}: Array of spectral window numbers, shape (Nspws)}
-\item{\textbf{telescope\_name}: Name of telescope. e.g. HERA. String.}
-\item{\textbf{time\_array}: Array of calibration solution times, center of integration, shape
-    (Ntimes), units Julian Date}
-\item{\textbf{time\_range}: Time range (in JD) that gain solutions are valid
-    for. list: [start\_time, end\_time] in JD.}
-\item{\textbf{x\_orientation}: Orientation of the physical dipole corresponding
-    to what is labelled as the x polarization. Values are east (east/west
-    orientation), north (north/south orientation) or unknown.}
+\item{\textbf{CALSTYLE}: \emph{string} Style of calibration. Possible values are ``sky'' or ``redundant''. (cal\_style)}
+\item{\textbf{CALTYPE}: \emph{string} Calibration type parameter. Possible values are ``delay'', ``gain'', or ``unknown''. (cal\_type)}
```

Comment:
While the UVCal object can have a cal_type of "unknown", I don't think a calfits file can. Partly because the format of the file depends on the cal_type. I think it errors if you try to write a UVCal object with cal_type="unknown" to a calfits file.